### PR TITLE
Custom theming

### PR DESCRIPTION
**Pull request by thecoolestguy**
Figured out a way to call overriding CSS from file on API backend server.  For discussion and/or proof of concept...

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 23c9014131f2d64ed5da1ed3f4000687afd8354c

https://deploy-preview-2874--ilios-frontend.netlify.com

File: app/templates/head.hbs [hunk 1]
```diff
@@ -6,3 +6,4 @@
     {{/each}}
   {{/if}}
 </title>
+<link rel="stylesheet" href="/school-theming/css/school.css">
```
Comment (thecoolestguy) [inline app/templates/head.hbs]: To test this change when proxying from your local system, prefix the href with 'https://ilios3-demo.ucsf.edu' so the line reads:

```<link rel="stylesheet" href="https://ilios3-demo.ucsf.edu/school-theming/css/school.css">```

**Comment by jrjohnson** [comment]
I believe we can do this without frontend modification.  Will put some notes in the API PR.

**Comment by thecoolestguy** [comment]
I've removed the call to the external CSS file on ilios3-demo.ucsf.edu, so if you check out this branch and re-build, you will no longer see the changes reflected in your local instance

**Pull request closed without merging**